buildSrc/src/main/java/com/uber/okbuck/core/model/base/TargetCache.java

```diff
@@ -52,7 +52,13 @@ public TargetCache(Project project) {
         case KOTLIN_LIB:
           targets =
               ImmutableMap.of(
-                  JvmTarget.MAIN, new JvmTarget(project, JvmTarget.MAIN, "kapt", "kaptTest"));
+                  JvmTarget.MAIN,
+                  new JvmTarget(
+                      project,
+                      JvmTarget.MAIN,
+                      "kapt",
```

Comment:
Should we have these kapt string in an Enum instead of hard coding them?

Reply:
You're right, but probably out of the scope of this PR to replace all occurrences of KAPT for the String